tests/Exclusion/Check/Fixture/docblock-on-private-property.php.inc

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Rector\Core\Tests\Rector\Exclusion\Check\DocBlockOnPrivateProperty;
+
+
+final class SomeController
+{
+    private $unusedProperty = [];
+
+    /**
+     * @noRector \Rector\DeadCode\Rector\Property\RemoveUnusedPrivatePropertyRector
+     */
+    private $anotherUnusedProperty = [];
+
+}
+
+?>
+    -----
+<?php
+
+namespace Rector\Core\Tests\Rector\Exclusion\Check\DocBlockOnPrivateProperty;
+
+
+final class SomeController
+{
+    /**
+     * @noRector \Rector\DeadCode\Rector\Property\RemoveUnusedPrivatePropertyRector
+     */
+    private $anotherUnusedProperty = [];
+
+}
+
+?>
```

Comment:
What's benefit of this class over the `fixture3291_keep_private_property.php.inc`? I think it can be removed.

It's better to have 1 test covering 1 issue.

Reply:
The is no benefit as I see, just understood that you wanted it like that. I can remove it.